### DISCUSSION TITLE
Can I hide a connector?

### DISCUSSION THREAD

**Discussion by iainlane** [dexidp/dex#3145] [Q&A]
I'm putting Dex in front of Argo Workflows, so that we can have human and machine users. Human users come from Google using an OAuth app (this works). Machine users are GitHub Actions workflows using OIDC (this also works since dcf7b18510d6ffc80c06c3bf655cdf3e05b8ca1d, at least as far as getting a token; I haven't tried to give it to Argo WF yet).

For this, I've got two connectors set up. One for each class of user. But the thing I don't know how to do is hide the connector selection screen. Currently, we're presented with a choice of Google or GitHub Actions. Only Google is a valid choice - I'd like to hide GHA and just send users straight to Google to sign in (like with `alwaysShowLoginScreen: false`).

But I can't see how to hide a connector in the UI. Is it possible?

**Comment by nabokihms** [comment]
There is no such feature now, but it can be implemented in the future.
